language-plutus-core/untyped-plutus-core/Language/UntypedPlutusCore.hs

```diff
@@ -0,0 +1,5 @@
+module Language.UntypedPlutusCore
```

Comment:
Module prefixes don't correspond to packages, that sounds like a Java-ism ;)

I don't object to `Language.PlutusCore.Typed` vs `Language.PlutusCore.Untyped`, I'd be happy with either.

Reply:
> Do we really want the untyped stuff in a separate package?

The package is the same, only the folder is different and module prefixes are different.

So far we've been mostly abbreviating Typed Plutus Core as Plutus Core and always talking about Untyped Plutus Core without abbreviating it, hence the naming decision. I don't mind restructuring things, but I'd rather do that in a different PR then.

Reply:
It's probably sensible to keep it in a separate folder during development since it'll be easier to find related files.  Do whatever you want though!  It's not vitally important.